config/routes.rb

```diff
@@ -6,11 +6,17 @@
       get "/auto_login", to: "users#auto_login"
       get "/users", to: "users#index"
       get "/user/:id", to: "users#show"
+
       resources :motorcycles, only: %i[index]
       get "/motorcycle/:id", to: "motorcycles#show"
       post "/motorcycle", to: "motorcycles#create"
       put "/motorcycle/:id", to: "motorcycles#update"
       delete "/motorcycle/:id => false", to: "motorcycles#destroy"
+
+      resources :reservations, only: %i[index]
+      get "/motorcycle/reservation/:id", to: "reservations#show"
+      post "/motorcycle/reservation", to: "reservations#create"
+      delete "/motorcycle/reservation/:id => false", to: "reservations#destroy"
```

Comment:
- [ ] Please  update this line  us mentioned before
```ruby
delete "/motorcycle/reservation/:id => false"
```
to 
```ruby
delete "/reservation/:id, to: to: "reservations#destroy"
```
  - Because it makes the routing impossible to access